### PR TITLE
making "active" status (of currency) less specific

### PR DESCRIPTION
**Pull request by ttodua**
In my experience, the "status" of currency is not a unified stance across exchanges, sometimes coin might have 'active' status for deposit, but not withdrawal (happens every day on all exchanges, when they list a new token, they allow withdrawals only after specific timespan, typically a day or more, to keep liquidity for a while), or sometimes both (withdrawal & deposit can be available, like it is for GYEN token on binance) but trading is not available.  Thus the 'status' of currency is a combination of different factors, depending exchange API (some exchanges provide separate fields for deposit/withdrawal/trading statuses, and some exchanges just provide only 1 property named 'status/etc' which you cant figure out what it means exactly - trading or funding or what delisted state.

Thus I think, we should be less specific with that property, and just use "or" relativity among those factors. When that property is to 'false' then user should take caution and make further research whether that currency is just disabled for trading, or deposit or withdrawal disabled only.

### CODE REVIEW

**Comment by kroitor** [comment]
Somehow, new branches still carry the edits from the other branches. It might cause conflicts upon merging one after another, in cases where i'm adding minor edits to the previous PRs.

File: wiki/Manual.md [hunk 1]
```diff
@@ -435,7 +435,7 @@ Below is a detailed description of each of the base exchange properties:
 
 - `timeout`: A timeout in milliseconds for a request-response roundtrip (default timeout is 10000 ms = 10 seconds). You should always set it to a reasonable value, hanging forever with no timeout is not your option, for sure.
 
-- `rateLimit`: A request rate limit in milliseconds. Specifies the required minimal delay between two consequent HTTP requests to the same exchange. The built-in rate-limiter is enabled by default and can be turned off by setting the `enableRateLimit` property to false.
+- `rateLimit`: A request rate limit in milliseconds. Specifies the required minimal delay between two consequent HTTP requests to the same exchange. The built-in rate-limiter is disabled by default and is turned on by setting the `enableRateLimit` property to true.
```
Comment (kroitor) [inline wiki/Manual.md:438]: If possible, let's rebase this PR cause it reverts some unrelated lines.

File: wiki/Manual.md [hunk 2]
```diff
@@ -676,10 +676,10 @@ If you encounter DDoS protection errors and cannot reach a particular exchange t
 - run your software in a distributed network of servers
 - ...
 
-# Markets
+# <a id="markets">Instruments & Markets</a>
```
Comment (kroitor) [inline wiki/Manual.md:679]: I would rather keep the initial title here, cause there are various links to it all over the place. If we do change titles here, we should also change links here: https://github.com/ccxt/ccxt/wiki (ccxt.wiki.git)

Comment (ttodua) [reply wiki/Manual.md:679]: The reason I've implemented the `<a id="markets">...` tag was to retain that and avoid breaking. So, the link will stay same nevertheless the title change. Am I wrong?

Comment (kroitor) [reply wiki/Manual.md:679]: I don't think it will work with the RTD, check this other comment: https://github.com/ccxt/ccxt/pull/10857#discussion_r767178164

Comment (ttodua) [reply wiki/Manual.md:679]: Ah I see. you mentioned `<a>` tags to break RTD.

Comment (ttodua) [reply wiki/Manual.md:679]: If so, then let's be title same as it was.
